docs/dev/README.md

```diff
@@ -13,4 +13,6 @@ This is not recommended if developing for only Armada.
 We suggest using this if you want to integrate armada with an external service like airflow.
 
 1) Run `docs/dev/setup_local_host.sh`
-2) Run each of those commands in separate terminals
\ No newline at end of file
+2) Run each of those commands in separate terminals
+
+Note: Currently due to End-to-End test rules on the current localdev setup, only the personal-anonymous user can by default have jobs created by anonymous user when using no authentication.
```

Comment:
This document isn't really describing the localdev setup -- so maybe this fact is not needed. I'd say move the change to docs/developer.md